docs/azmcp-commands.md

```diff
@@ -187,6 +187,12 @@ azmcp keyvault key get --subscription <subscription> --vault <vault-name> --key
 azmcp keyvault key create --subscription <subscription> --vault <vault-name> --key <key-name> --key-type <key-type>
 ```
 
+### Azure Service Bus Operations
+```bash
+# Peeks at messages in a Service Bus queue.
```

Comment:
None of the other comments end with  punctuation, we should be consistent (all or none)